deployment/README.md

```diff
@@ -24,6 +24,12 @@ $ cd deployment
 $ pip install -r deployment/requirements.txt
 ```
 
+## S3 Website Failover
+
+Route 53 DNS failover is configured by the following deployments steps in the event that all application servers fail simultaneously. The contents of the failover site are located in `src/failover_website` from the project root.
+
```

Comment:
The text mentions "configured by the following deployment steps" but there are no steps listed after the paragraph.